packages/proto/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "./lib",
```

Comment:
This conflicts with our clean script:

```
        const { outDir } = tsconfig.compilerOptions;
        if (outDir) {
          const outPath = resolve(packagePath, outDir);
          console.log(`Removing ${outPath}`);
          rmSync(outPath, { recursive: true, force: true });
```

We shouldn't mix generated code with normal code.